static/app/views/performance/newTraceDetails/traceDrawer/details/styles.tsx

```diff
@@ -107,13 +107,6 @@ function EventDetailsLink(props: {eventId: string; projectSlug?: string}) {
   );
 }
 
-type DurationComparisonProps = {
-  baseline: number | undefined;
-  totalDuration: number;
-  isComparingSelfDuration?: boolean;
```

Comment:
@Abdkhan14 in most cases like these, you should always pass "type" and not "isXXX" because. It is more future proof to extending and I find it to be slightly more explicit